### DISCUSSION TITLE
NodeViewContent default value?

### DISCUSSION THREAD

**Discussion by cusxio** [ueberdosis/tiptap#2584] [General Questions]
When using `NodeViewContent` is there a way to set a default value in it?

```jsx
<NodeViewWrapper as="figure" data-drag-handle>
  <img alt={alt} src={src} />
  <NodeViewContent as="figcaption" />
</NodeViewWrapper>
```

I'm trying to pre-fill the `figcaption` with some text.

**Comment by philippkuehn** [comment]
I don’t think this is possible right now. But you can try to use CSS pseudo elements to render some text if the figcaption is empty.
